### PR TITLE
Alert on deprecated controls in Winforms

### LINKED ISSUE #782: ApiAlertAnalyzer to flag Deprecated Menu Controls
All removed controls are listed [here](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fdotnet%2Fcore%2Fcompatibility%2Fwinforms%23removed-controls&data=04%7C01%7Csubal%40microsoft.com%7C743d5268f21f4897e90d08d95aebe05a%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637640791433032205%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=6bVx6FzbjeDZ68u53lDmNAIY9UFG1VkmozfV7UWeN5w%3D&reserved=0) and [here](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fdotnet%2Fcore%2Fcompatibility%2Fwindows-forms%2F5.0%2Fwinforms-deprecated-controls&data=04%7C01%7Csubal%40microsoft.com%7C743d5268f21f4897e90d08d95aebe05a%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637640791433042201%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=%2BgXLmtVfpO0RstilPJkEy4NizxVz7C6eDNoI5t2v20g%3D&reserved=0)

### PR DESCRIPTION
**Pull request by sunandabalu**
Fixes 
#782
#783

### CODE REVIEW

**Comment by sunandabalu** [comment]
@RussKie FYI, this is the change to alert on deprecated controls, could you review the accuracy of categorizations?

**Review by twsouthwick — APPROVED** [review]
I can't really comment here as to the correctness of the change, but I'll approve it so once you get that from a SME, you can merge :)

**Review by RussKie — COMMENTED** [review]
I don't have any understanding how it works, so can't really comment on correctness or completeness. The changeset looks sensible and in line what's been discussed over emails.

File: NuGet.config
Comment (twsouthwick) [inline NuGet.config:14]: @Sweekriti91 I'm pretty sure we can't do this. What package are you needing here?

Comment (Sweekriti91) [reply NuGet.config:14]: Ohhhh that's was for maui-check. Okay let me remove maui-check and I'll add the dotnet commands the maui-check does under the hood instead. I'll remove this.

Comment (sunandabalu) [reply NuGet.config]: You can get maui-check added to one of the public azdo feeds you have listed and remove nuget.org as a source.

**Comment by twsouthwick** [comment]
@sunandabalu Until the maui stuff is worked out, why don't you just skip those tests and @Sweekriti91 can handle that separately so this can get merged it

File: tests/tool/Integration.Tests/IntegrationScenarios/WpfSample/vb/Upgraded/WpfApp1/WpfApp1.vbproj
Comment (twsouthwick) [inline tests/tool/Integration.Tests/IntegrationScenarios/WpfSample/vb/Upgraded/WpfApp1/WpfApp1.vbproj:22]: @sunandabalu You'll probably need to undo these changes as well

**Pull request merged**
